content/blog/2019/11/2019-11-28-do-plugins-store-credentials-in-a-secure-way.adoc

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Do Plugins Store Credentials In A Secure Way? - DevOps World | Jenkins World 2019"
+tags:
+- jenkins world
+- jenkinsworld
+- devopsworld2019
+- security
+authors:
+- woodspeed
+opengraph:
+  image: /images/post-images/PluginWebForm.png
```

Comment:
```suggestion
  image: /images/post-images/PluginWebForm.png
```

Please create a new directory and put the image inside it.
The name is too generic, and it does not fit well the root folder (which is not the recommended practice anyway)